app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
```

Comment:
Will this return a users name or the email itself?

Reply:
Just the email, this is a devise generated template.